test/test_utils/__init__.py

```diff
@@ -350,6 +353,14 @@ def is_time_for_canary_safety_scan():
     current_utc_time = time.gmtime()
     return current_utc_time.tm_hour == 16 and (0 < current_utc_time.tm_min < 20)
 
+def is_time_for_invoking_ecr_scan_failure_routine_lambda():
+    """
+    Canary tests run every 15 minutes.
+    Using a 20 minutes interval to make tests run only once a day around 9 am PST (10 am during winter time).
+    """
+    current_utc_time = time.gmtime()
+    return current_utc_time.tm_hour == 16 and (0 < current_utc_time.tm_min < 20)
```

Comment:
Please address this comment in a follow-up PR, so that we can unblock the merge of this PR and make this feature usable.

You can merge this function with the `is_time_for_canary_safety_scan` function, and just make it a more generic time-of-day gate function, with a configurable time-of-day when you want to allow some repetitive activity to be allowed, in both the safety check as well as ecr scan.